charts/radar-mockserver/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+name: radar-mockserver
+version: "0.1.0"
+appVersion: "5.15.0"
+description: Chart to deploy MockServer to Kubernetes. MockServer enables easy mocking of any system you integrate with via HTTP or HTTPS with clients written in Java, JavaScript and Ruby.  MockServer also includes a proxy that introspects all proxied traffic including encrypted SSL traffic and supports Port Forwarding, Web Proxying (i.e. HTTP proxy), HTTPS Tunneling Proxying (using HTTP CONNECT) and SOCKS Proxying (i.e. dynamic port forwarding).
+home: "http://mock-server.com"
+sources:
+  - "https://github.com/mock-server/mockserver"
+  - "https://hub.docker.com/r/mockserver/mockserver"
+maintainers:
+  - email: pim@thehyve.nl
+    name: Pim van Nierop
+    url: https://www.thehyve.nl/experts/pim-van-nierop
+  - email: nivethika@thehyve.nl
+    name: Nivethika Mahasivam
+    url: https://www.thehyve.nl/experts/nivethika-mahasivam
+icon: http://mock-server.com/apple-touch-icon.png
+dependencies:
+  - name: mockserver
+    version: 5.15.0
+    repository: https://www.mock-server.com
```

Comment:
Ideally we want to mirror this chart in our own repository but I don't know how much it's necessary

Reply:
Can we make a dependency on a chart in the _external_ dir?

Reply:
Ah apparently we can:

```
# Chart.yaml
dependencies:
- name: nginx
  version: "1.2.3"
  repository: "file://../dependency_chart/nginx"
```

Reply:
We have done it like this in the past
https://github.com/RADAR-base/radar-helm-charts/blob/main/charts/velero-s3-deployment/Chart.yaml#L11

I'm not sure if the `file://` approach will work but if it does, it's fine then.